services/catalog/requirements/_test.in

```diff
@@ -17,9 +17,11 @@ pytest-cov
 pytest-mock
 pytest-runner
 pytest-docker
+docker-compose==1.26.2 # avoids increasing via pytest-docker
```

Comment:
I would remove that dependency, i.e. fix pytest-docker version so it does not bring docker-compose in there.

Reply:
same as above